ShowerShow/Model/User.cs

```diff
@@ -15,8 +15,8 @@ public class User
         [JsonRequired]
         public string PasswordHash { get; set; }
         public string Email { get; set; }
-        public List<UserFriend> Friends { get; set; }
-        public List<Achievement> Achievements { get; set; }
+        public List<UserFriendDTO> Friends { get; set; } = new List<UserFriendDTO>();
+        public List<Achievement> Achievements { get; set; } = new List<Achievement>();
```

Comment:
Why new list in the model? Just curious/confused

Reply:
For the Automapping purposes.